client/src/app/containers/Home/useActiveGames.ts

```diff
@@ -21,6 +21,7 @@ export function useActiveGames(): ReturnType {
     const { currentUser } = useAuth();
     const { setAlert } = useAlert();
     const dispatch = useDispatch();
+    const base = `https://lambda.hobbycribbage.com`;
```

Comment:
At the very least, we might pull this out into a separate file so it can be shared in all the places it needs to be. Perhaps we could make a `cribbageClient` of some sort.

Reply:
:bulb: Right now, these are all the same because we're using lambdas for everything. Someday, we could have a "games" service, and a "recommendation" service, and a "AI hand identifier" service. In each of those, they might hit different subdomains.

Perhaps, it's best to have a `gameBaseURL` var that we would use for all of the current usages...

Reply:
I'm fine with that. We could also say YAGNI and keep it `baseURL` until we need something different, but my main concern is centralizing the value of it, not the identifier.

💡 🔮 Maybe it would be fun to learn about OpenAPI/Swagger and get it to generate a TypeScript SDK for us based on our `games` API spec